src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/ChatManager.java

```diff
@@ -153,4 +154,8 @@ private void registerHandlers() {
                 filter(allyPlayer -> allyPlayer.toPlayer() != null).
                 collect(Collectors.toList());
     }
+
+    private boolean discordSRVExists() {
```

Comment:
This could be public (for using in a condition, see other comment).
This name is a bit misleading, it doesn't mention the hook being enabled.

Reply:
> This could be public (for using in a condition, see other comment). This name is a bit misleading, it doesn't mention the hook being enabled.

I didn't think up another name 😃 🔫

Reply:
Could be isDiscordHookEnabled
Not sure